x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -188,13 +219,30 @@ function NodeDetailLink({ name, item }: { name: string; item: ProcessTableView }
           )}
         </EuiBadge>
       ) : (
-        <>
+        <StyledButtonTextContainer>
           <CubeForProcess
             running={!isTerminated}
+            isOrigin={isOrigin}
             data-test-subj="resolver:node-list:node-link:icon"
           />
-          <span data-test-subj="resolver:node-list:node-link:title">{name}</span>
-        </>
+          <StyledProcessLabel color={descriptionText}>
+            {isOrigin && (
+              <div data-test-subj="resolver:node-list:node-link:origin" className="origin-label">
+                <span>
+                  {i18n.translate(
+                    'xpack.securitySolution.endpoint.resolver.panel.table.row.originLabel',
+                    {
+                      defaultMessage: 'Analyzed Event',
+                    }
+                  )}
+                </span>
+              </div>
+            )}
+            <div data-test-subj="resolver:node-list:node-link:title">
+              <span>{name}</span>
```

Comment:
Is this `span` doing anything useful?

Reply:
It was when I went a different route for maintaining the ellipse, but not needed anymore. Taken out